src/modules/app_event_subscribers/autologin.js

```diff
@@ -0,0 +1,91 @@
+import { subscribeEventReceived } from '../appEvents';
+import { sendAppCommands } from '../sendAppCommands';
+import { broadcastEvent } from '../app_commands/broadcastEvent';
+import { sendTrustedPipelineRequest } from '../app_commands/sendPipelineRequest';
+import { sendHttpRequest } from '../app_commands/sendHttpRequest';
+
+const AUTOLOGIN_PIPELINE = 'bigcommerce.user.autologin.v1';
+const CURRENT_CUSTOMER_SERIAL = 'current_customer';
+
+let autologinProcessFinished = false;
+
+/**
+ * Requests current customer data.
+ */
+function requestCurrentCustomer() {
+  // This is client id of our bigcommerce app.
+  // It is not an oauth client id generated from the store control panel.
+  const clientId = 'ked0n332n5c8opplowtqmc6kb817wot';
+  sendAppCommands([
+    sendHttpRequest({
+      url: `https://store-r5s844ad.mybigcommerce.com/customer/current.jwt?app_client_id=${clientId}`,
+      serial: CURRENT_CUSTOMER_SERIAL,
+      method: 'GET',
+    }),
+  ], '13.0');
```

Comment:
Why do you set this version to '13' ? Does it mean only newer apps will be able to execute this command? How do we handle the other older apps?

Reply:
I've checked with Pascal, the versions fro the switch between legacy and Shopgate PWA atm are:
iOS codebase  >= 10.15.0
Android >= 5.18.0
I believe that what they'll do eventually is just increase version switch for Android as the Shopgate PWA has already dependency towards lib version 13.0 [see](https://github.com/shopgate/pwa/search?utf8=%E2%9C%93&q=httprequest&type=)
Also what will be done in Api Login/Logout story for the BigCommerce will rely on the same code base.